src/lib/plot/axis/x-axis.js

```diff
@@ -31,14 +32,21 @@ const propTypes = {
   ])
 };
 
+delete propTypes.axisType;
```

Comment:
Here's a related discussion: https://groups.google.com/forum/#!topic/v8-users/zE4cOHBkAnY
and https://github.com/google/google-api-nodejs-client/issues/375#issuecomment-76538779 .

For example with rest: `const {axisType, ...propsWithoutAxisType} = Axis.propTypes;`